### PR TITLE
Newsweek clone built with Bootstrap 4

### PR DESCRIPTION
**Pull request by SajjadAhmad14**
This web project is inspired by the design of Newsweek.com. It's built with HTML, CSS, and Bootstrap4.
Bootstrap Grid System has been used to layout web pages for three viewports.
Bootstrap spacing utilities have been used for margin and padding.
Bootstrap navbar classed have been utilized to layout navigation bars.

### CODE REVIEW

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
### Status: Chnages required :recycle: 

### Hi @SajjadAhmad14  :hand: 

### Your project is looking good, but there is some changes to be made:
- Yours are using STickler in your project, You have to use Github Actions instead, go to https://github.com/microverseinc/linters-config/tree/master/html-css and flow the necessary steps.
- In medium widths, the navbar is overlapping over section titles.
clone:
Screenshot from 2020-06-28 23-13-55
original:
Screenshot from 2020-06-28 23-15-26
- There are some images that are broken:
Screenshot from 2020-06-28 23-16-07
Screenshot from 2020-06-28 23-17-17
Screenshot from 2020-06-28 23-17-25
- In large widths ( over 2000 px) your nav bar expand and the page stay fixed, please fix that:
clone
Screenshot from 2020-06-28 23-28-39
original
Screenshot from 2020-06-28 23-30-30

### Once you are done with the changes, please ask for another code review.

**Review by elmejdki — CHANGES_REQUESTED** [review]
# Status: require changes :recycle: 
## Great job so far guys :clap:  :clap:  :clap:, your clone almost matches the original but in order to get it approved you need to make some changes :hammer: :hammer_and_wrench: :wrench: :toolbox: 

- [ ] center the text inside the navbar:
### clone: :point_down: :point_down:  :point_down:  :point_down:  :point_down: 
image
 ### original: :point_down: :point_down:  :point_down:  :point_down:  :point_down: 
image

- [ ] Remove border-bottom on the last article, and make the images wider:
### clone: :point_down: :point_down:  :point_down:  :point_down:  :point_down: 
image
### original: :point_down: :point_down:  :point_down:  :point_down:  :point_down: 
image

- [ ] Fix the subscribe and newsletter section, it should look similar to the original:
### clone: :point_down: :point_down:  :point_down:  :point_down:  :point_down: 
image

### original: :point_down: :point_down:  :point_down:  :point_down:  :point_down: 
image

- [ ] change the size of the logo to match the original on table view:
### clone:  :point_down: :point_down:  :point_down:  :point_down:  :point_down: 
image
### original: :point_down: :point_down:  :point_down:  :point_down:  :point_down: 
image

- [ ] Fix the form bellow opinion section on tablet view:
### clone: :point_down: :point_down:  :point_down:  :point_down:  :point_down: 
image
### original: :point_down: :point_down:  :point_down:  :point_down:  :point_down: 
image

### Please make all the required changes before you request another code review.
## Happy Coding :tophat:

**Review by rammazzoti2000 — CHANGES_REQUESTED** [review]
### Hello @SajjadAhmad14 :wave: I am Alex your code reviewer for this project
You did great job on this project, good use of bootstrap, readme file descriptive although is showing only when I click on it and not the root directory of the repo.

### :stop_sign: Required Changes :construction: :construction_worker: 
- [ ] HTML code doesn't pass validation please use [this validator ](https://validator.w3.org/#validate_by_input) by direct input for more information
image
- [ ] Your sections are not using labels in the bottom left as the original page, some are showing but only text without a background while some are totally missing, please compare your page with the original one for better comparison
### Orignal
image
### Clone
image


### Large Screens
- [ ] The `Subscribe` & `Newsletter` sections are missing some contrast like background color and also some border on `Newsletter` section. As for the footer the logo is way too big and the social icons are not the same color, and same goes for the elements inside `Subscribe` & `Newsletter`, please compare your page against the original one for better comparison.
### Original
image
### Clone
image
- [ ] You can notice when resizing the page around 1008px the navbar logo gets smaller for no reason
image

### Medium Screens
- [ ] You can notice this advertisement elements along with `signup` card should be center and the elements inside `signup` card are too big check you page against the original one for better comparison
### Original
image
### Clone
image
- [ ] The `Subscribe` section is not the same as the original, check out bellow :point_down: 
### Original
image
### Clone
image

### Small Screens
- [ ] The navbar logo is quite big
image
- [ ] The `My Turn` & `Culture and Travel` sections are not stretching on the entire width of the screen
### Original
image
### Clone
image
- [ ] The articles from each specific section (Business, US, World etc.) have this huge gap between the thumbnail and title description
image
- [ ] `Subscribe` and `Newsletter` need some work, please compare your page against the original one
### Original
image
image

### Clone
image
image

- [ ] The footer also needs some adjustments
image

### <em>Kindly fix the REQUIRED issues and send another code review request.

Cheers and Happy coding! :clap: 

Code review by Alexandru Bangau

(Feel free to contact me on slack by the handle of **_alexandru_** if you have any additional question)</em>
### :stop_sign: Status: Required Changes

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Nice work so far. 👍 
## However, there are some changes required 👀 🧰 🔨 🔧 

- The original page has spacing on the sides. Your version is missing the gray space on the sides of the page. Please add this spacing to your version.

| Your version | Original page |
| --- | --- |
| image | image |

- Your version is missing the LOGIN link on the Medium Display.

| Your Version | Original page |
| --- | --- |
| image | image |

- The side thumbnails from the bottom articles appear stretched. Please fix this.

image

- The Subscribe and Newsletter sections have different heights at some points while resizing the page. Please make sure that they have the same height where it corresponds. Also, the phone image from the Newsletter section appears stretched. Make sure the aspect ratio is preserved while resizing.

| Your version | Original page |
| --- | --- |
| image | image |

### Submit these changes and request a new code review! 😄 👍 
### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🛑

## Hello @SajjadAhmad14 👋. You have done a great job making this project 💯 ! 

### However, we still need to work on some changes.

- Your README file has a typo(`RAEDME`) in its filename. That's why it's not showing in your repo.
image

- Increase the margin on both sides of your clone.

- Please fix the navbar.

#### Clone
image

#### Original
image

- Decrease the size of the images in this section 👇
image

#### Original
image

- Please make this look like original 👇

#### Clone

> Desktop

image

>Mobile

image


#### Original

> Desktop

image

> Mobile

image


Apply these changes and request another review.


Happy coding 👨‍💻

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,475 @@
...
+  box-sizing: border-box;
+}
+
+.container-fluid {
+  font-family: "Roboto Condensed", sans-serif;
+}
+
+.container {
+  font-family: "Roboto Condensed", sans-serif;
+}
```
Comment (andresporras3423) [inline assets/css/style.css]: Group selectors:
`
.container-fluid, .container {
  font-family: "Roboto Condensed", sans-serif;
}
`
Or create a new class:
`
.<new_class_name> {
  font-family: "Roboto Condensed", sans-serif;
}
`

File: assets/css/style.css [hunk 2]
```diff
@@ -0,0 +1,475 @@
...
+  color: rgba(3, 3, 3, 0.4);
+}
+
+.card a {
+  color: #000;
+}
+
+.more a {
+  color: #000;
+}
```
Comment (andresporras3423) [inline assets/css/style.css]: Group selectors, or create a new class

File: assets/css/style.css [hunk 3]
```diff
@@ -0,0 +1,475 @@
...
+
+.nav-2 a:hover {
+  color: #f72210;
+  text-decoration: none;
+}
+
+.nav-3 a:hover {
+  color: #f72210;
+  text-decoration: none;
```
Comment (andresporras3423) [inline assets/css/style.css]: Group selectors or create a new class.

File: assets/css/style.css [hunk 4]
```diff
@@ -0,0 +1,475 @@
...
+
+.debate a:hover {
+  color: #f92859;
+  text-decoration: none;
+}
+
+.debate-2 a:hover {
+  text-decoration: none;
+  color: #f92859;
+}
```
Comment (andresporras3423) [inline assets/css/style.css]: group selectors, or create a new class

File: assets/css/style.css [hunk 5]
```diff
@@ -0,0 +1,475 @@
...
+
+#debate a:hover {
+  color: rgba(3, 3, 3, 0.4);
+  text-decoration: none;
+}
+
+#opinion a:hover {
+  color: rgba(3, 3, 3, 0.4);
+}
```
Comment (andresporras3423) [inline assets/css/style.css]: Group selectors, or create a new class. Don't overuse ID selectors.

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @SajjadAhmad14** 
Good job with your new submission for this project :muscle:  
Your page really looks like the original :clap: :clap: :clap:  
Your page passes most of the requirements. There are just a few details to improve before you can go ahead:
- [ ] Don't overuse ID Selectors
- [ ] Don't repeat yourself, in order to minimize code you can group selectors with the same style definition, or you can create a new class.

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Review by elmejdki — APPROVED** [review]
# status: approved :heavy_check_mark: 
## Great job guys your clone looks really good :clap: :100: :1st_place_medal:  , thank you for making the required changes.

- now you can merge to master your project is approved.

## Happy Coding :tophat:

**Pull request merged**